.github/workflows/unpublish.yml

```diff
@@ -0,0 +1,26 @@
+name: Unpublish
+
+on: [pull_request]
```

Comment:
This should be triggered manually, not automatically triggered as part in response to any event